src/frontend/js/components/TeacherDashboardCourseList/index.spec.tsx

```diff
@@ -102,7 +102,7 @@ describe('components/TeacherDashboardCourseList', () => {
     );
```

Comment:
in commit msg: something like: 

`useUnionResources` do not use react-query cache but it's own custom cache mechanic.
This custom mechanic didn't handle the refetch part of react-query.
There a update that handle queries refetch, as it should be, in our custom useUnionResource hook.

Reply:
I will provide more context